packages/components/src/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -198,6 +207,32 @@ export const ConfirmationModal = forwardRef(function ConfirmationModalInternal(
     });
   }, [title, message, confirmLabel, cancelLabel, onConfirm, onCancel]);
 
+  const handleConfirm = () => {
+    dispatch({ type: "confirm" });
+    onRequestClose && onRequestClose();
+  };
+
+  const handleCancel = () => {
+    dispatch({ type: "cancel" });
+    onRequestClose && onRequestClose();
+  };
+
+  useEffect(() => {
+    const confirmShortcut = createKeyDownHandler("Enter", handleConfirm);
+    const cancelShortcut = createKeyDownHandler(
+      ["Escape", { metaKey: true, key: "." }],
+      handleCancel,
+    );
+    if (open || state.open) {
+      window.addEventListener("keydown", confirmShortcut);
```

Comment:
How would you feel about the idea of using a `useOnKeyDown` hook for these?

We have one here that could be enhanced and used. It would be even better to split out into our hooks package.
https://github.com/GetJobber/atlantis/blob/master/packages/components/src/Autocomplete/Menu.tsx#L160-L174

If you're by chance looking for another PR that could actually be a great opportunity as well. 😄